### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by bmreading** [gtk-rs/gtk-rs.github.io#271]
I don't see a license for this project. I guess this should be added?

**Comment by sdroege** [comment]
Is there any specific reason why you need a license? You want to take parts of the website to integrate into something else?

But you're right of course, it would be useful to have a license for the website too. Just not sure what makes most sense for this. CC BY-SA? Technically we would also have to check with everybody who ever contributed to this repository.

@bilelmoussaoui @GuillaumeGomez ?

**Comment by bmreading** [comment]
> Is there any specific reason why you need a license? You want to take parts of the website to integrate into something else?

No, nothing like that. The only thing that occurred to me was possibly using the logo images for use in sites like Wikipedia, where licensing matters, and possibly enablement for people to utilize it for swag like stickers or something. Both would require a relatively liberal license that allowed for commercial usage with allowed derivatives.

**Comment by sdroege** [comment]
I think that's the license of the logo, but @bilelmoussaoui should be able to confirm that.

**Comment by GuillaumeGomez** [comment]
By default we should use the same license as the other gtk-rs projects.
